server/src/contact/contact.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm';
+
+@Entity('contact')
+export class Contact {
+  @PrimaryColumn()
+  contactid: string;
+
+  @Column()
+  emailaddress1: string;
+
+  @Column()
+  fullname: string;
+
+  @Column()
+  statecode: string;
+
+  @Column()
+  statuscode: string;
+}
```

Comment:
Again, this is a TypeORM object, which is specifically meant for modeling SQL database objects. There is no SQL database for this project. We wouldn't even need to use this.